conf/clusters/prod.json

```diff
@@ -77,7 +77,7 @@
       ]
     }
   },
-  "enable_threat_intel": false,
+  "enable_threat_intel": true,
```

Comment:
I may miss some information, can you remind me why do we want to enable threat intel by default?

Reply:
@chunyong-lin for the "integration" test fixtures for threat intel to work, this needs to be enabled. do you envision this being a problem? I can disable the rule if so

Reply:
just pushed commits that disable this rule by default, commenting as to why